packages/focusjail/src/FocusJailContainer.spec.tsx

```diff
@@ -161,6 +161,45 @@ describe('FocusJailContainer', () => {
         expect(focusSpy).toHaveBeenLastCalledWith(container);
       });
 
+      it("doesn't intercept tab key if not the first or last tabbable item", () => {
+        const { getByTestId } = render(
+          <BasicExample
+            focusElem={() => undefined}
+            restoreFocus={false}
+            focusableChildren={
+              <>
+                <p>non-focusable test</p>
+                <button>First button</button>
+                <button>Second button</button>
+                <button>Last button</button>
+              </>
+            }
+          />
+        );
+
+        const focusJailContainer = getByTestId('container');
+        const firstButton = screen.getByRole('button', { name: /First button/iu });
```

Comment:
Does `getByText` work here? If we can avoid `regex` usage that would be awesome.

Reply:
The `getByRole` API provides some confidence in terms of a11y which `getByText` does not (particularly for interactive elements). I think `getByText` would be more suitable for  `<div>hello</div>`, `<span>hello</span>`, etc. Using `getByRole` with a regex was mostly inspired by examples from https://kentcdodds.com/blog/common-mistakes-with-react-testing-library.